packages/js/product-editor/src/components/index.ts

```diff
@@ -9,7 +9,7 @@ export { DetailsNameField as __experimentalDetailsNameField } from './field-deta
 export { DetailsFeatureField as __experimentalDetailsFeatureField } from './field-details-feature';
 export {
 	DetailsCategoriesField as __experimentalDetailsCategoriesField,
-	CategoryField as __experimental__experimentalDetailsCategoryField,
+	CategoryField as __experimentalDetailsCategoryField,
```

Comment:
Wait, isn't this still wrong? Shouldn't it be `__experimentalCategoryField`?

Having something named `__experimentalDetailsCategoriesField` and another called the nearly identical `__experimentalDetailsCategoryField` is super confusing.

I'm on the fence as to whether to export `CategoryField` at all. Since it is being exported as `__experimental` though, I suppose we can export it and always change our mind later.

Reply:
Whoops again! I'm a little sleep deprived today 😓 .

I went ahead and removed the export for now. We can re-add it if we feel there's a reason.